launcher/src/renderer/components/ReplayInfoDialog.tsx

```diff
@@ -36,10 +36,17 @@ export default function ReplayInfoDialog({
 
   React.useEffect(() => {
     (async () => {
-      const proc = spawn(app, "replaydump", [
-        path.join(getReplaysPath(app), filename),
-        "dump-ewram",
-      ]);
+      const proc = spawn(
+        app,
+        "replaydump",
+        [
+          path.join(getReplaysPath(app), filename),
+          "dump-ewram",
+        ],
+        {
+          env: { ...process.env, ...env },
```

Comment:
Hooray, I'm learning TypeScript 😅

By the way, rather than set it for each use of `spawn` individually, would it be possible to just configure this in `process.ts`, so it applies to all usages?

I assume this is either overridden or not the same thing, but before getting this merged I thought I'd ask:

https://github.com/tangobattle/tango/blob/30ab0442a74f26084c6bcdef38ccc21ec2da8665/launcher/src/process.ts#L37

Reply:
Alrighty then, I'll fix the stuff I broke 👍